README.md

```diff
@@ -88,6 +88,22 @@ Here are the keys that vmfloaty currently supports:
 - url
   + String
 
+### Tab Completion
+
+There is a basic completion script for Bash (and possibly other shells) included with the gem in the [extras/completions](https://github.com/briancain/vmfloaty/blob/master/extras/completions) folder. To activate, that file simply needs to be sourced somehow in your shell profile.
+
+For convenience, the path to the completion script for the currently active version of the gem can be found with the `floaty completion` subcommand. This makes it easy to add the completion script to your profile like so:
+
+```bash
+source $(floaty completion --shell bash)
+```
+
+If you are running on macOS and use Homebrew's `bash-completion` formula, you can symlink the script to `/usr/local/etc/bash_completion.d/floaty` and it will be sourced automatically:
+
+```bash
+ln -s $(floaty completion --shell bash) /usr/local/etc/bash_completion.d/floaty
+```
```

Comment:
Hrm. Shortcoming: if you've installed `floaty` as a gem or via the Homebrew tap I smashed together last year, this won't work.


```
mckern@flexor vmfloaty (git:add_bash_completion) $ (set -x; floaty completion --shell bash)
+ floaty completion --shell bash
mckern@flexor vmfloaty (git:add_bash_completion) $ 
```

`extras/completions/floaty.bash` isn't actually installed anywhere right now by the gem, so it can't be referenced. Thoughts?